types/react-native/index.d.ts

```diff
@@ -9731,7 +9732,8 @@ declare global {
     /**
      * This variable is set to true when react-native is running in Dev mode
      * Typical usage:
```

Comment:
Isn't this now obsolete with the explicit `@example`?

Reply:
Yes, it was and I have now removed it:

<img width="398" alt="Screen Shot 2021-12-10 at 16 14 39" src="https://user-images.githubusercontent.com/34488374/145648501-b01a14c0-3cb5-461b-b050-8831b8382a4c.png">